### ISSUE TITLE
Usage for multiple flickity instances

### ISSUE THREAD

**Issue by chrisjansky** [elcontraption/flickity-transformer#10]
Hey!

Is it possible to initialise flickity-transformer for multiple flickity elements on the same page? Using the code below, only the last instance found has the transformer properly applied.

```javascript
var coverflowEl = document.querySelectorAll("[data-coverflow]");

coverflowEl.forEach(function(instance, index) {
  var flkty = new Flickity(instance, {
    prevNextButtons: false,
    pageDots: false
  });

  var flktyTransform = new FlickityTransformer(flkty, [
    {
      name: 'scale',
      stops: [
        [-100, 0.5],
        [0, 1],
        [100, 0.5]
      ]
    }
  ])
})
```

**Comment by elcontraption** [comment]
Thanks for catching this! It appears we have a variable scope issue. Will try to fix later today.

**Comment by elcontraption** [comment]
Should be fixed with v0.3.6.
